lib/tasks/write_deploy_json.rake

```diff
@@ -1,10 +1,11 @@
 namespace :login do
   desc 'generate a generic deploy.json file'
-  task :deploy_json do
+  task deploy_json: :environment do
```

Comment:
This seems to be causing issues for the build. What was it added for?

Reply:
Cuz Rubocop wants that.  I'm working on it.  Thanks for the continuous attention; apologies for not pushing faster... multitasking.